src/components/tl/mlx5/mcast/tl_mlx5_mcast.h

```diff
@@ -33,6 +33,14 @@
 #define DROP_THRESHOLD    1000
 #define MAX_COMM_POW2     32
 
+/* Allgather RDMA-based reliability designs */
+#define ONE_SIDED_RELIABILITY_MAX_TEAM_SIZE 1024
+#define ONE_SIDED_NO_RELIABILITY            0
+#define ONE_SIDED_SYNCHRONOUS_PROTO         1
+#define ONE_SIDED_ASYNCHRONOUS_PROTO        2
+#define ONE_SIDED_SLOTS_COUNT               2                /* number of memory slots during async design */
```

Comment:
what do we call design here?

Reply:
please look at https://github.com/openucx/ucc/pull/980#discussion_r1624895422